Files/ViewModels/Bundles/BundleItemViewModel.cs

```diff
@@ -146,6 +165,18 @@ private async void SetIcon()
             {
                 try
                 {
+                    if (Path.EndsWith(".lnk"))
+                    {
+                        var (IconData, OverlayData, IsCustom) = await associatedInstance.FilesystemViewModel.LoadIconOverlayAsync(Path, 24u);
+
+                        await CoreApplication.MainView.ExecuteOnUIThreadAsync(async () =>
+                        {
+                            Icon = await IconData?.ToBitmapAsync();
```

Comment:
It's my bad sorry:
`await IconData?.ToBitmapAsync();` -> wrong, it will crash since you can't `await null`
`await IconData.ToBitmapAsync();` -> ok, since `ToBitmapAsync` checks for null

Reply:
No worries 😅